app/models/form_submission.rb

```diff
@@ -0,0 +1,13 @@
+# frozen_string_literal: true
+
+class FormSubmission < ApplicationRecord
+  has_kms_key
+  has_encrypted :form_data, key: :kms_key, **lockbox_options
+
+  has_many :form_submission_attempts, dependent: :destroy
+  belongs_to :in_progress_form, optional: true
+  belongs_to :saved_claim, optional: true
+  belongs_to :user_account, optional: true
```

Comment:
Is there a reason to make the `user_account` association optional?

Reply:
Yes, Simple Forms handles unauthenticated submissions.